### DISCUSSION TITLE
Nous SZ-T04 NOT supported Zigbee2MQTT version 1.25.0-dev c

### DISCUSSION THREAD

**Discussion by erwan33fr** [Koenkk/zigbee2mqtt#12227] [Devices]
Hello,

I have by this device on Aliexpress :
https://fr.aliexpress.com/item/1005003324613168.html

![Capture](https://user-images.githubusercontent.com/86933399/164774973-67fc1a9a-5d1c-4126-9770-5d67ca5cf18f.JPG)

It seems to be part of the supported device :
https://www.zigbee2mqtt.io/devices/SZ-T04.html

When i add the device I have this error message :
Zigbee2MQTT:warn  2022-04-20 15:09:09: Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_c7emyjom'

I use Zigbee2MQTT version 1.25.0-dev commit: 7e61252d
Coordinator type zStack3x0
Coordinator revision 20210708
Frontend version 0.6.80
Stats
Total 30
By device type
Router: 22
End devices: 8

**Comment by erwan33fr** [comment]
How to modify the source code of the device "[Nous SZ-T04](https://www.zigbee2mqtt.io/devices/SZ-T04.html)" to allow support for device with Zigbee model 'TS0601' and manufacturer name '_TZE200_c7emyjom' which seems to be exactly the same ?

**Reply by Koenkk** [reply]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by erwan33fr** [reply]
Perfect thank you very much.
The device is well recognized and supported now :-)

**Reply by unmesh59** [reply]
@Koenkk 

Has this been added to 1.27? That's the version I'm running from within HA 

Thanks

**Reply by Koenkk** [reply]
`_TZE200_c7emyjom` is supported in 1.27.0

**Comment by unmesh59** [comment]
@erwan33fr 

I've been looking for a humidity sensor with a display and clock and this looks very promising. How often does this device provide sensor updates?

Thanks

**Reply by erwan33fr** [reply]
every 0.8° change because that's what I set.
![02](https://user-images.githubusercontent.com/86933399/184940361-3f4d7ef9-6363-483d-b40b-e7fd5801ec2d.JPG)

![01](https://user-images.githubusercontent.com/86933399/184940085-a0097e10-67ac-4387-829c-326994f3125f.JPG)

**Reply by erwan33fr** [reply]
I also tested this sensor which works very well also with zigbee2mqtt : 
https://fr.aliexpress.com/item/1005003980647546.html

But this one reports the temperature at regular time intervals (adjustable via zigbee2mqtt) and also according to the temperature change (but without being able to adjust the sensitivity via zigbee2mqtt).

![03](https://user-images.githubusercontent.com/86933399/184941089-c38e92f9-2bf4-4c48-a92d-eba74942d04d.JPG)

**Reply by erwan33fr** [reply]
In terms of battery longevity, I think the "Nous SZ-T04" is better.
And it has a built-in magnet that makes it easy to mount on a metal surface.
But the time display is written a bit small for my eyes.

**Reply by unmesh59** [reply]
Thanks for all the information.

What about the humidity reporting intervals or sensitivity?

How would you compare the two units' display quality in terms of how well the numbers stand out? My eyesight isn't what it used to be either and the pictures of the first one look better on the AliExpress website!

And can you say more about the battery life differences you've observed?

**Comment by unmesh59** [comment]
Thanks

**Comment by Daw34** [comment]
Witam, mam podobny czujnik jest to chyba podróbka lecz nie działa. https://sklep.automat-on.pl/Czujnik-temperatury-i-wilgotnosci-Zigbee-Tuya.html


{"id":2,"type":"EndDevice","ieeeAddr":"0xa4c138eae9e06ab0","nwkAddr":8545,"manufId":4417,"manufName":"_TZE200_vaarscrt","powerSource":"Battery","modelId":"TS0601","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":81,"inClusterList":[4,5,61184,0],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"65503":"V�l+\u0011","65506":56,"65508":0,"stackVersion":0,"dateCode":"","appVersion":72}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":72,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1675222365778,"defaultSendRequestWhen":"immediate"}


Zgłasza błąd:
Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Warning: Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_vaarscrt'

**Reply by gex64** [reply]
I've same problem
